ourhexenv.py

```diff
@@ -376,7 +381,6 @@ def render(self, mode="human"):
         pygame.display.flip()
         self.clock.tick(30)
 
-
```

Comment:
Let's follow PEP 8 and have 2 empty lines between each function. 
We're also supposed to have 2 empty lines between each class, but we're already doing that, so that's fine.

Reply:
Ran autopep8 on the env file.